### ISSUE TITLE
node key duplication

### ISSUE THREAD

**Issue by eiDear** [dbeaver/cloudbeaver#1669]
**Describe the bug**
When I visit https://demo.cloudbeaver.io/，I get error ”Node Key Duplication“

**To Reproduce**
1. Run Browser, and enter "https://demo.cloudbeaver.io/"
2. And then this error will pop in  the left-bottom screen.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser 华为浏览器 版本
 - Version 12.1.3.301 | 内核版本: 99.0.4844.84

**Additional context**
Add any other context about the problem here, e.g. error log.
```plaintext
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/array_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/bpchar_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/char_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/cidr_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/date_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/float4_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/float8_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/inet_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/int2_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/int4_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/int8_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/interval_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/macaddr_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/macaddr8_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/name_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/numeric_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/oid_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/oidvector_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/text_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/time_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/timestamptz_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/timetz_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/varchar_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/timestamp_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/bool_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/bytea_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/abstime_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/text_pattern_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/varchar_pattern_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/bpchar_pattern_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/reltime_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/uuid_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/pg_lsn_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/enum_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/tsvector_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/tsquery_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/range_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/box_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/jsonb_ops
database://postgres-jdbc-17542a12bf1-98e1dc6705767f0/org.jkiss.dbeaver.ext.postgresql.model.PostgreInformation/org.jkiss.dbeaver.ext.postgresql.model.PostgreAccessMethod/btree/org.jkiss.dbeaver.ext.postgresql.model.PostgreOperatorClass/citext_ops
```

**Comment by TatyanaSsau** [comment]
@eiDear this is a known issue. 
And thank you for the report. 
We are going to upgrade demo (https://demo.cloudbeaver.io/#/) soon. And this error will be fixed

**Comment by TatyanaSsau** [comment]
@eiDear We have been updated to a new version!
https://demo.cloudbeaver.io/#/ is available with 23.1.0 version
